usaspending_api/etl/es_etl_helpers.py

```diff
@@ -309,7 +314,7 @@ def put_alias(client, index, alias_name, award_type_codes):
 def create_aliases(client, index, silent=False):
     for award_type, award_type_codes in INDEX_ALIASES_TO_AWARD_TYPES.items():
         alias_name = "{}-{}".format(settings.TRANSACTIONS_INDEX_ROOT, award_type)
-        if silent is False:
+        if silent is True:
```

Comment:
Why does this print when silent is true? I would assume that variable suppresses logs.

Reply:
This is going to be a merge conflict I think.  This was fixed.

Reply:
This needs to go back to `if silent is False:`.